cmd/podman/common/netflags.go

```diff
@@ -26,6 +26,13 @@ func DefineNetFlags(cmd *cobra.Command) {
 	)
 	_ = cmd.RegisterFlagCompletionFunc(addHostFlagName, completion.AutocompleteNone)
 
+	baseHostsFileFlagName := "base-hosts-file"
+	netFlags.String(
+		baseHostsFileFlagName, "",
+		`Path to a hosts file to copy the entries into the container, or one of the special values. ("image"|"none")`,
```

Comment:
I don't like it copying in, it should replace the entries.  The values should be 

"" (Default), "none"==--no-hosts, /PATHTO/hosts file.